Carmunication/Controllers/SessionList/GroupAdd/GroupAddViewController.swift

```diff
@@ -12,7 +12,7 @@ import SnapKit
 final class GroupAddViewController: UIViewController {
 
     var groupDataModel: Group = Group()
-    var pointsDataModel: [Point2] = []
+    var pointsDataModel: [Point] = []
```

Comment:
빈 배열을 표현할 때는 요렇게들 많이 사용하더라구요
`var 변수명: 타입 = 값` 보다는 `var 변수명 = 값` 처럼 타입을 꼭 써줘야 하는 경우가 아니면 생략하는 경우가 많습니다

```suggestion
    var pointsDataModel = [Point]()
```

Reply:
> 빈 배열을 표현할 때는 요렇게들 많이 사용하더라구요 `var 변수명: 타입 = 값` 보다는 `var 변수명 = 값` 처럼 타입을 꼭 써줘야 하는 경우가 아니면 생략하는 경우가 많습니다

저도 수정해야겠군요😮